### PR TITLE
Fix doc for sensitivity thresholds

### PR DESCRIPTION
**Pull request by obrix**
Fix doc for sensitivity thresholds

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java [hunk 1]
```diff
@@ -94,7 +94,8 @@ public SensitivityAnalysisParameters setVoltageVoltageSensitivityValueThreshold(
 
     /**
      * VoltageVoltageSensitivityValueThreshold is the threshold under which sensitivity values having
-     * variable type BUS_TARGET_VOLTAGE and function type BUS_VOLTAGE will be filtered from the analysis results.
+     * variable type BUS_TARGET_VOLTAGE
```
[the remark this answers is not in the document]
Comment (annetill) [reply sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java:97]: No no, no mistake here indeed for me.

File: sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java [hunk 2]
```diff
@@ -109,7 +110,7 @@ public SensitivityAnalysisParameters setFlowVoltageSensitivityValueThreshold(dou
     /**
      * FlowVoltageSensitivityValueThreshold is the threshold under which sensitivity values having
      * variable type among INJECTION_REACTIVE_POWER and BUS_TARGET_VOLTAGE
-     * and function type among BUS_VOLTAGE, BRANCH_REACTIVE_POWER_1/2/3 and BRANCH_CURRENT_1/2/3
```
Comment (annetill) [inline sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java:112]: No error here too, maybe just some clarifications...

**Comment by annetill** [comment]
@obrix do you want to clarify something finally?

**Comment by obrix** [comment]
@geofjamg @annetill The issue was that the the couple BUS_TARGET_VOLTAGE/BUS_VOLTAGE is documented as filtered by flowVoltage threshold and voltageVoltageThreshold, where in reality only voltageVoltageThreshold filters those kind os sensitivity values. Do you prefer another way to modify the documentation from the one I used or do you feel it is already explicit enough without modification ?

File: sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java (hunk 2 above)
Comment (annetill) [reply sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java:112]: FlowVoltageSensitivityValueThreshold is the threshold under which sensitivity values having variable type among INJECTION_REACTIVE_POWER and function type among BUS_VOLTAGE or variable type among BUS_TARGET_VOLTAGE and function type among BRANCH_REACTIVE_POWER_1/2/3 and BRANCH_CURRENT_1/2/3.
Is it better for you?

Comment (obrix) [reply sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisParameters.java:112]: yes!

**Pull request merged**